docs/versioning.rst

```diff
@@ -0,0 +1,156 @@
+**********
+Versioning
+**********
+
+
+Current Release Stages
+======================
+
+.. list-table:: Releases
+   :widths: 20 30 30 20
+   :header-rows: 1
+
+   * - Release
+     - Runtimes
+     - Minimum Trace Agent
+     - Stage
+   * - 0.x
+     - CPython: 2.7, 3.5–3.10
+     - 
```

Comment:
we should define a minimum, we can pick an old 6.x version or can even just check what the oldest version any customer is currently using and use that. what do you think?